systemd/ceph-exporter.target

```diff
@@ -0,0 +1,8 @@
+[Unit]
+Description=ceph target allowing to start/stop all ceph-exporter@.service instances at once
```

Comment:
@peng225 and this one?

Reply:
@tchaikov Sorry again.
I realized that `ceph-exporter.target` existed to start/stop all `ceph-exporter@.service` instances at once.
Now that we have removed `@`, I believe there is no reason to have this file.
I deleted the file and updated `ceph-exporter.service.in` and `CMakeLists.txt` accordingly.